### ISSUE TITLE
Option to disable kubectl download

### ISSUE THREAD

**Issue by jakolehm** [lensapp/lens#489]
**What would you like to be added**:

Option to disable kubectl download. In this case Lens could fallback to bundled kubectl version.

**Why is this needed**:

Download servers are not always accessible (firewalls, av softaware etc.).

Related issues #202 , #395

**Comment by Nokel81** [comment]
Work is done, parked on https://github.com/Nokel81/lens/tree/kubectl-download-config 

Waiting for the react refactor PR to land.

**Comment by jakolehm** [comment]
@Nokel81 I think it would be ok to open a draft PR.

**Comment by Nokel81** [comment]
Sounds good, I will initially target it against `vue_react_migration` and then once that lands, change it to `master`

**Comment by nevalla** [comment]
Let's fix this with #679
